distribution/feature-packs/server-feature-pack/pom.xml

```diff
@@ -72,6 +72,16 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
Did you check if this dependency is available in Brew? If not we'll need to build it in Brew before we can merge so it doesn't break RH-SSO builds

Reply:
How do I check the Brew stuff?  I know nothing about Brew.